client/card-readers/list/index.js

```diff
@@ -27,12 +29,51 @@ const ReadersListDescription = () => (
 );
 
 const ReadersList = () => {
+	const { readers, isLoading } = useReaders( 10 );
+
 	return (
 		<SettingsLayout displayBanner={ false }>
 			<SettingsSection Description={ ReadersListDescription }>
 				<LoadableSettingsSection numLines={ 20 }>
 					<Card className="card-readers-list__wrapper">
-						<CardBody>List</CardBody>
+						<CardBody className="card-readers-list__header">
+							<div className="card-readers-list__header-id">
+								{ __( 'Reader ID', 'woocommerce-payments' ) }
+							</div>
+							<div className="card-readers-list__header-model">
+								{ __( 'Model', 'woocommerce-payments' ) }
+							</div>
+							<div className="card-readers-list__header-status">
+								{ __( 'Status', 'woocommerce-payments' ) }
+							</div>
+						</CardBody>
+						<CardDivider />
+						<CardBody size={ null }>
+							<ul>
+								{ ! isLoading &&
+									Object.entries(
+										readers
+									).map( ( [ index, reader ] ) => (
+										<CardReaderListItem
+											key={ index }
+											reader={ reader }
+										/>
+									) ) }
+							</ul>
+						</CardBody>
+						{ /*
```

Comment:
We can throw away the dead code for now and restore from history later when needed.

Reply:
Dead code removed.